Chapters/2-1-Chapter8.md

```diff
@@ -0,0 +1,159 @@
+## Chapter 8
+
```

Comment:
Why is Liffa's section in first person, and this in third? ::disappointed::

Reply:
we talked about this before but i forgot. I wanted Liffa to be the narrator for everything.

Reply:
choosing third person limited to get those thoughts in there without having to explain that Liffa is the narrator.